tigerbeetle-hs.cabal

```diff
@@ -1,4 +1,5 @@
-cabal-version:      3.0
+cabal-version:   3.0
```

Comment:
I ran `cabal-fmt`, it might be nice to use that, but can reset the diff if you want.